src/Factories/DIRuleFactory.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @link      http://www.yiiframework.com/
+ *
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license   http://www.yiiframework.com/license/
+ */
+
+namespace Yiisoft\Rbac\Factories;
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Rbac\Rule;
+
+/**
+ * @author Andrii Vasyliev <sol@hiqdev.com>
```

Comment:
- `@author` мы больше не прописываем. По git blame видно. План удалить отовсюду потому как отвлекает от сути когда читаешь.
- `@since` нужен будет только после первого релиза. Тоже можно удалить.